tests/gold_tests/autest-site/microserver.test.ext

```diff
@@ -114,15 +140,16 @@ def MakeOriginServer(obj, name, port=False, ip=False, delay=False, ssl=False, op
         ip = '127.0.0.1'
     if (delay == False):
         delay = 0
-    command = "python3 {0} --data-dir {1} --port {2} --ip_address {3} --delay {4} -m test --ssl {5}".format(
-        server_path, data_dir, port, ip, delay, ssl)
+    command = "python3 {0} --data-dir {1} --port {2} --ip_address {3} --delay {4} -m test --ssl {5} --lookupkey {6}".format(
```

Comment:
check why the command invoked is different from this command

Reply:
Hahaha, interesting way to solve the `-` vs. `_` problem :-).  For this, I'd look at putting the argument in quotes to avoid possible shell expansions. E.g. `--lookupkey '{6}'`.